lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -117,6 +117,7 @@ describe('FilePickerApp', () => {
         },
         null /* groupSet */
       );
+      assert.isTrue(wrapper.exists('FullScreenSpinner'));
```

Comment:
It might be useful to add 
`assert.isFalse(wrapper.exists('FullScreenSpinner'));` above selectContent in this test to show its hidden first, then renders. Same for the test below.

Alternately, we could just add new base test that ensures the spinner is not loaded if no action has been taken.

Reply:
Thanks for the feedback.  I have extracted the checks for the `FullScreenSpinner` into separate tests and added checks that the spinner is not displayed when the form is initially rendered.